File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] value = null;
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    value = new int[] {i, j};
                    break;
                }
            }
        }
        return value;
    }
}

File: Data Structures & Algorithms/binary-search/submission-0.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        left=0
        right=len(nums)-1
        index=-1
        while left<=right:
            mid=(left+right)//2
            if (list_v := nums[mid]) == target:
                index=mid
                break
            elif list_v>target:
                right=mid-1
            else:
                left=mid+1
        return index